template/project.yml

```diff
@@ -1,6 +1,8 @@
 project:
   # This will be used as the vendor name in the new project's composer.json file.
   vendor_name: 'acquia'
+  # This will be used as the description in the new project's composer.json file.
+  description: 'A short description of the package. Usually this is just one line long.'
```

Comment:
After the new project is created, this value will still be in the project.yml but it won't do anything. If someone changes the composer.json file, it will diverge from the value here. We've got the same issue with vendor_name. 

We could change the comments to be less about project generation and simply allow these values to be here for anyone reading the file. 

Thoughts?